opencensus/trace/ext/grpc/server_interceptor.py

```diff
@@ -11,37 +11,71 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import collections
+import logging
 
 import grpc
-import logging
 
 from opencensus.trace import attributes_helper
-from opencensus.trace import tracer as tracer_module
+from opencensus.trace import tracer as tracer_module, execution_context
```

Comment:
Change this to two lines to import the `tracer` module and `execution_context` module separately.